plugins/ProfessionalServices/tests/UI/ProfessionalServices_PluginPromo_spec.js

```diff
@@ -96,4 +96,20 @@ describe("ProfessionalServices_PluginPromo", function () {
         expect(await page.screenshotSelector('.pluginPromo')).to.matchImage('promo_sessionrecordings');
     });
 
+    it('can dismiss a promo and no longer see it in menu', async function() {
+        const category = 'ProfessionalServices_PromoFormAnalytics';
+        const subcategory = 'ProfessionalServices_PromoOverview';
+
+        await page.goto(urlBase + 'category=' + category + '&subcategory=' + subcategory);
+        await page.waitForNetworkIdle();
+
+        let promoSessionRecordingMenuItemCount = await page.evaluate(() => $('li.menuTab[data-category-id=ProfessionalServices_PromoFormAnalytics]').length);
```

Comment:
Should we use the `const category` defined above here in the menu item selector instead of hardcoding it in the string?

Reply:
Could do but then need to pass it in to the function. I'll leave it for now.